bin/upgrade/dev/scripts/member_permission_generator.js

```diff
@@ -0,0 +1,52 @@
+var pluginManager = require('../../../plugins/pluginManager.js'),
+    async = require('async'),
+    countlyDb = pluginManager.dbConnection();
```

Comment:
Now this method returns promise and not db object, since we got rid of mongoskin